src/main/java/com/hivemq/configuration/service/impl/listener/InternalListenerConfigurationService.java

```diff
@@ -1,60 +0,0 @@
-/*
- * Copyright 2019-present HiveMQ GmbH
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package com.hivemq.configuration.service.impl.listener;
-
-import com.google.common.collect.ImmutableList;
-import com.hivemq.extension.sdk.api.annotations.NotNull;
-import com.hivemq.configuration.service.entity.Listener;
-
-/**
- * An internal subinterface with extension for the {@link ListenerConfigurationService}
- *
- * @author Dominik Obermaier
- */
-public interface InternalListenerConfigurationService extends ListenerConfigurationService {
```

Comment:
was this not used before?

Reply:
No, this was not used other from some basic unit tests for it. 
Side effects are not expected since it is not used 🤷 .